spec/lib/virus_scanner_spec.rb

```diff
@@ -1,4 +1,4 @@
-require 'spec_helper'
+require 'rails_helper'
```

Comment:
Minor: I'd prefer this orthogonal change to be in a separate commit, but thanks for fixing it!